applications/nrf5340_audio/include/nrf5340_audio_common.h

```diff
@@ -53,4 +56,19 @@ struct bt_mgmt_msg {
 	struct bt_le_ext_adv *ext_adv;
 };
 
+/*** Messages regarding volume ***/
+enum volume_evt_type {
+	VOLUME_INVALID,
+	VOLUME_UP,
+	VOLUME_DOWN,
+	VOLUME_SET,
+	VOLUME_MUTE,
+	VOLUME_UNMUTE,
+};
+
+struct volume_msg {
+	enum volume_evt_type event;
+	uint8_t value;
```

Comment:
The naming is fine, but we should consider changing to 
```suggestion
	uint8_t volume;
```